extensions/keycloak-authorization/deployment/src/test/resources/offline-enforcer-application.properties

```diff
@@ -0,0 +1,14 @@
+# Configuration file
+quarkus.oidc.client-id=backend-service
+quarkus.oidc.credentials.secret=secret
+quarkus.oidc.tls.verification=none
```

Comment:
@michalvavrik Is tls related config (and the keystore) needed here ?

Reply:
It's a leftover, I removed it. Originally I tried to create much better test based on QuickStarts stuff, but I never managed to make it work.